marc_to_solr/lib/traject_config.rb

```diff
@@ -691,6 +691,7 @@
 to_field 'case_file_notes_display', extract_marc('5653abcde')
 to_field 'methodology_notes_display', extract_marc('567a')
 to_field 'editor_notes_display', extract_marc('570a') # added
+to_field 'action_notes_display', extract_marc('583a') # added
```

Comment:
Per Mark S., need to limit this to notes created by Princeton (subfield 5 should have 'NJP)

Should also make sure not to record Private notes.